contrib/docker/builder/Dockerfile

```diff
@@ -8,8 +8,22 @@ RUN add-apt-repository ppa:bitcoin/bitcoin
 RUN apt-get update && apt-get install -y \
     autoconf automake bsdmainutils ccache cmake curl g++ g++-mingw-w64-x86-64 gcc gcc-mingw-w64-x86-64 git \
     libboost-all-dev libbz2-dev libcap-dev libdb4.8-dev libdb4.8++-dev libevent-dev libminiupnpc-dev libprotobuf-dev \
-    libqrencode-dev libssl-dev libtool libzmq3-dev make pkg-config protobuf-compiler python-pip qtbase5-dev \
-    qttools5-dev-tools python3-zmq
+    libqrencode-dev libssl-dev libtool make pkg-config protobuf-compiler python-pip qtbase5-dev \
+    qttools5-dev-tools python3-zmq zlib1g-dev build-essential uuid-dev libsodium-dev
```

Comment:
I can't find `uuid` and `libsodium` in `depends/packages`. What I mean is it will cause CI to success but fail when release building.

Reply:
These packages are just dependancies to build `libzmq` from source, afterward they are not necessary. Is there a way I can test locally?

Reply:
They are both just dependancies for building zmq from source. However `uuid` is actually not necessary, and `libsodium-dev` is not needed if we compile with `--without-libsodium`. I have updated the Dockerfile for this